### PR TITLE
docs: start adding docs

### PR DESCRIPTION
**Pull request by chee**
This adds some very basic documentation. we'll need to fill this out more as we progress

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
+# Contributing
+
+Hey, so you want to contribute to Origami.
+
+
+
+## Before you get started
```
Comment (chee) [inline CONTRIBUTING.md:7]: wtf typora

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+| --------- | ---------------------------------------------------------------------------------------------------------------------------------------------------- |
+| feat      | a feature has been added. this translates to a minor release.                                                                                        |
+| fix       | a bug has been patched. this translates to a patch release.                                                                                          |
+| docs      | a change to documentation. these commits do not cause a release, but the messages appear in the "Documentation" section of the next release's notes. |
+| backstage | any changes that aren't user-facing. no release is caused, and these messages will not appear in release notes.                                      |
+
+Indicate a breaking change by placing an `!` between the type name and the colon, e.g.:
+
+```
+feat!: add large mouse
```
Comment (JakeChampion) [inline CONTRIBUTING.md:31]: 🐭

**Review by JakeChampion — APPROVED** [review]
This is good, I like the tips section a lot, I often forget what the command looks like to run something in a specific component

**Comment by chee** [comment]
> This is good, I like the tips section a lot, I often forget what the command looks like to run something in a specific component

we should add a little script that people can install. `npm i -g ./tools/origami-helper` and then they can be like `oh test o-tooltip`

**Pull request merged**